tests/ci/sync-to-aws.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -euo pipefail
+
+source /tmp/.env
+
+[ -z "$IMAGE_KEY" ] && error "Could not find '$IMAGE_KEY', qcow image has not been created"
+DOWNLOAD_DIRECTORY="/var/lib/libvirt/images"
+S3_BUCKET_NAME="fedora-testing-farm-image-import"
+AWS_CLI="aws"
+
+# Configure aws region and output format
```

Comment:
Code comments should be used only when the code is not self explanatory or we want to explain the why behind the piece of code.

Reply:
Done